lib/core-server/src/build-static.ts

```diff
@@ -68,6 +70,24 @@ export async function buildStaticStandalone(options: CLIOptions & LoadOptions &
     ...options,
   });
 
+  const staticDirs = await presets.apply<StorybookConfig['staticDirs']>('staticDirs');
+
+  if (staticDirs && options.staticDir) {
+    throw new Error(`Conflict when trying to read staticDirs:
+    * Storybook's configuration option: 'staticDirs'
+    * Storybook's CLI flag: '--staticDir' or '-s'
+     
+    Choose one of them, but not both.
+    `);
```

Comment:
Can you use `ts-dedent` here? This way the output in the CLI is properly aligned!
```suggestion
    throw new Error(dedent(`Conflict when trying to read staticDirs:
    * Storybook's configuration option: 'staticDirs'
    * Storybook's CLI flag: '--staticDir' or '-s'
     
    Choose one of them, but not both.
    `));
```

Reply:
I didn't know this library. Thanks for sharing :)